libs/docs/platform/message-popover/message-popover-docs.component.html

```diff
@@ -0,0 +1,81 @@
+<fd-docs-section-title id="default" componentName="message-popover">
+    Message Popover with Simple forms
+</fd-docs-section-title>
+<description>
+    <p>Message popover component supports both template-driven and reactive forms.</p>
+    <p>
+        To define human-readable form item label text, developers need to use
+        <code>fdpMessagePopoverFormItem</code> directive.
```

Comment:
Does it mean that we cannot specify error type and heading/descriptionwith this approach? Is it intended?

Reply:
@platon-rov developers can define it on module level with `withConfig` method, or provide it on component level with `FDP_MESSAGE_POPOVER_CONFIG ` token. We support 2 options: plain string (will be threatened as a heading only), and more complex object which contains header text, description text and error type